assets/app/view/create_game.rb

```diff
@@ -44,26 +44,45 @@ def render_content
     end
 
     def render_inputs
+      @min_p = {}
+      @max_p = {}
+
       games = (Lib::Params['all'] ? Engine::GAMES : Engine::VISIBLE_GAMES).map do |game|
+        @min_p[game.title] = game::CERT_LIMIT.keys.min
+        @max_p[game.title] = game::CERT_LIMIT.keys.max
+
         title = game.title
         title += " (#{game::GAME_LOCATION})" if game::GAME_LOCATION
-        if game::DEV_STAGE == :production
-          h(:option, { attrs: { value: game.title } }, title)
-        else
-          h(:option, { attrs: { value: game.title } }, "#{title} (#{game::DEV_STAGE})")
-        end
+        title += " (#{game::DEV_STAGE})" if game::DEV_STAGE != :production
+
+        h(:option, { attrs: { value: game.title } }, title)
       end
-      h(:div, [
-        render_input('Game Title', id: :title, el: 'select', children: games),
-        render_input('Description', id: :description),
+
+      limit_range = lambda do
+        min = Native(@inputs[:max_players]).elm.min = @min_p[Native(@inputs[:title]).elm.value]
+        max = Native(@inputs[:max_players]).elm.max = @max_p[Native(@inputs[:title]).elm.value]
+        val = Native(@inputs[:max_players]).elm.value.to_i
+        Native(@inputs[:max_players]).elm.value = (val < min || val > max ? max : val)
```

Comment:
val.between?(min, max) or (min..max).incude?(val) probably need to check the end points

Reply:
I had used tried include, but ran into some string/integer probs (which I solved after changing, didn’t revert though). between? won’t work, will change it to include.